koku/masu/util/azure/common.py

```diff
@@ -186,10 +187,17 @@ def azure_generate_daily_data(data_frame):
     return data_frame
 
 
-def match_openshift_resources_and_labels(data_frame, cluster_topology, matched_tags):
+def match_openshift_resources_and_labels(data_frame, cluster_topologies, matched_tags):
     """Filter a dataframe to the subset that matches an OpenShift source."""
-    nodes = cluster_topology.get("nodes", [])
-    volumes = cluster_topology.get("persistent_volumes", [])
+    nodes = chain.from_iterable(cluster_topology.get("nodes", []) for cluster_topology in cluster_topologies)
+    volumes = chain.from_iterable(
+        cluster_topology.get("persistent_volumes", []) for cluster_topology in cluster_topologies
+    )
+    nodes = []
+    volumes = []
+    for cluster_topology in cluster_topologies:
+        nodes.extend(cluster_topology.get("nodes", []))
+        volumes.extend(cluster_topology.get("persistent_volumes", []))
     matchable_resources = nodes + volumes
```

Comment:
No need to do this twice.
```suggestion
    matchable_resources = list(nodes) + list(volumes)
```

Reply:
definitely an oversight on my part, thanks for catching that!